desktop.css

```diff
@@ -573,6 +573,13 @@
     resize: none;
```

Comment:
- [ ] [OPTIONAL] Please notice how the texts entered in the message field are hardly visible, I would advise to either remove the color property or change it to something dark and with good contrast on your `.forms-container form textarea`  so that users entering the message can see it clearly, 
![image](https://user-images.githubusercontent.com/97521557/211092140-87f944a4-3e1a-4be1-8a5c-d1c861061212.png)